lombok.config

```diff
@@ -0,0 +1,17 @@
+# Should work but IDEA plugin seemingly ignores it: https://github.com/mplushnikov/lombok-intellij-plugin/issues/759
+# import ../shared/lombok-shared.config
```

Comment:
Dunno if we want this type of reference, no matter how small.

Reply:
I'll remove it. Thanks :)